src/tools/perf/perftest.c

```diff
@@ -297,6 +301,21 @@ static ucs_status_t setup_sock_rte(struct perftest_context *ctx)
     int sockfd, connfd;
     int ret;
 
+    if (UCX_PERF_TEST_FLAG_LOOPBACK_CONN & ctx->params.super.flags) {
+        ctx->flags |= (TEST_FLAG_PRINT_TEST |
```

Comment:
When concatenating both the test and results, the formatting looks weird:
```
+------------------------------------------------------------------------------------------+
| API:          protocol layer                                                             |
| Test:         tag match latency                                                          |
| Data layout:  (automatic)                                                                |
| Send memory:  host                                                                       |
| Recv memory:  host                                                                       |
| Message size: 8                                                                          |
+--------------+--------------+------------------------------+---------------------+-----------------------+
|              |              |        latency (usec)        |   bandwidth (MB/s)  |  message rate (msg/s) |
+--------------+--------------+----------+---------+---------+----------+----------+-----------+-----------+
|    Stage     | # iterations | 50.0%ile | average | overall |  average |  overall |  average  |  overall  |
+--------------+--------------+----------+---------+---------+----------+----------+-----------+-----------+
Final:                 20000                         0.577                 264.65                34688508
```

What do you think of either (1) separating the test from the results with a blank line, or (2) making the "test" cell wider so that they align on the same width?

Reply:
Can we improve the output formatting here please?

Reply:
pls remove ( ) around (TEST_FLAG_PRINT_TEST | TEST_FLAG_PRINT_RESULTS);